miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/api/RetrofitCreatorUtils.kt

```diff
@@ -31,14 +32,20 @@ internal fun createRetrofitClient(
     @Suppress("SpreadOperator")
     val httpClient = OkHttpClient.Builder()
         .addHeaderInterceptor(*headers.asArray())
+        .addInterceptor(provideHeaderInterceptor())
         .build()
     return Retrofit.Builder()
-        .addConverterFactory(
-            GsonConverterFactory.create(
-                GsonBuilder().setLenient().create()
-            )
-        )
+        .addConverterFactory(GsonConverterFactory.create(GsonBuilder().setLenient().create()))
         .baseUrl(baseUrl)
         .client(httpClient)
         .build()
 }
+
+private fun provideHeaderInterceptor(): Interceptor = Interceptor { chain ->
+    val original = chain.request()
+    val request = original.newBuilder()
+        .header("Accept-Encoding", "identity")
```

Comment:
Was this header being automatically set to something else by OkHttp? Or it just wasn't being set at all?

Also, from what I see [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding), `identity` means that the server will never send a compressed response. The Mini App is downloaded as a compressed ZIP file anyways, so I don't think that's a big deal. 

But it's possible that the `listing` endpoint can return a very long list, so the request will take longer when uncompressed. I guess the listing endpoint doesn't need to be called very often, so probably it's not a big deal. What do you guys think?

Reply:
This header will be included on every api call request by OkHttp.

Yeah, there is not much impact on performance. For now, there are a lot of miniapps in the list but I don't see the difference. What do you think @khairul-alam-licon ?

Reply:
in fact this could lead to an unexpected high traffic when MiniApp will be popular. But this leaves us time to think about it later when the problem will be raised, if it is ever raised. This fix can be considered until then, with a comment maybe.

Reply:
Looks like "Accept-Encoding", "identity" here is overriding the `Accept-Encoding:gzip` provided from RAS, so we are losing the `gzip` compression feature. I could see a few miniapps in STG to trace the performance issue. I am not sure if we could use a testing api which provides enough miniapps to test the traffic. But, of course, we can address the performance with long list in future, in that case, adding a TODO comment would be better.

Reply:
We don't need the long list of miniapp to see the performance issue on sdk call side. Because if there is a long list of miniapps, the backend should do paging on that list.